### ISSUE TITLE
Typographical (English) errors in document page

### ISSUE THREAD

**Issue by dkenny** [MicrosoftDocs/sql-docs#8079]
My wife is a Technical Writer, so I notice these things:

The acronym  "SPN" does not begin with a vowel. Throughout this document page, there are consistent references to "an SPN". This is incorrect English, it should be "a SPN", since the noun object begins with a consonant, not a vowel. I wouldn't have gone through the effort and provided this feedback if it was just a single misplaced typo -- however, it is consistently displayed as "an SPN" multiple times throughout the document, and may also be pervasive in other pages that reference the acronym "SPN". As someone who has developed public-facing web pages since the early days of the world-wide web, I know that I very much appreciated when a user pointed out typographical errors to me, rather than just ignore it. I hope you find that same value in this feedback.
 
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 16136000-fc6d-9156-499a-c45bde264bc4
* Version Independent ID: ea9aacfc-8f27-563f-08c8-77ac4e941cff
* Content: [Register a Service Principal Name for Kerberos Connections - SQL Server](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/register-a-service-principal-name-for-kerberos-connections?view=sql-server-ver16)
* Content Source: [docs/database-engine/configure-windows/register-a-service-principal-name-for-kerberos-connections.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/database-engine/configure-windows/register-a-service-principal-name-for-kerberos-connections.md)
* Product: **sql**
* Technology: **configuration**
* GitHub Login: @rwestMSFT
* Microsoft Alias: **randolphwest**

**Comment by WilliamAntonRohm** [comment]
@dkenny -- thank you for your close reading.  It depends whether one pronounces the acronym "SPN" as "Spin" or "Ess-P-En" and it seems the latter is preferred (or more popular).  I appreciate your feedback!

**Comment by dkenny** [comment]
@WilliamAntonRohm -- thanks. Alas, I learned that I was completely off-base, my wife (technical writer) corrected me, and agreed that the proper article-label to use is "an SPN" base on the "Ess-pee" pronounciation, as in "an hour". It just nagged me each time I saw it as I read through the web article. Sorry about jumping into the feedback without proper research, I should have checked with her first!
